task/sast-unicode-check/0.2/MIGRATION.md

```diff
@@ -0,0 +1,6 @@
+# Migration from 0.1 to 0.2
+
+Version 0.2:
+
+- The `image-digest` parameter has been introduced back, to be used in ORAS uploading.
```

Comment:
Do the users need to add this parameter to their existing build pipelineRun definitions?

Reply:
yep, 
added the user's note